tools/docker-compose/README.md

```diff
@@ -427,3 +428,33 @@ Once the playbook is done running LDAP should now be setup in your development e
 4. awx_ldap_org_admin:orgadmin123
 
 The first account is a normal user. The second account will be a super user in AWX. The third account will be a system auditor in AWX. The fourth account is an org admin. All users belong to an org called "LDAP Organization". To log in with one of these users go to the AWX login screen enter the username/password. 
+
+
+### Splunk Integration
+
+Splunk is a log aggregation tool that can be used to test AWX with external logging integration. This section describes how to build a reference splunk instance and plumb it with your AWX for testing purposes.
+
+First, be sure that you have the awx.awx collection installed by running `make install_collection`.
+
+Next, install the splunk.es collection by running `ansible-galaxy collection install splunk.es`.
+
+Anytime you want to run a Splunk instance alongside AWX we can start docker-compose with the SPLUNK option to get a Splunk instance with the command:
+```bash
+SPLUNK=true make docker-compose
+```
+
+Once the containers come up three new ports (8000, 8090 and 9199) should be exposed and the Splunk server should be running on those ports. The first port (8000) is the non-SSL admin port and you can log into splunk with the credentials admin/splunk_admin. 
+
+Now we are ready to configure and plumb Splunk with AWX. To do this we have provided a playbook which will:
+* Backup and configure the External Logging adapter in AWX. NOTE: this will back up your existing settings but the password fields can not be backed up through the API, you need a DB backup to recover this.
+
+Before we can run the playbook we need to understand that Splunk will be communicated to from within the AWX container. Because of this, we have to tell AWX how to route traffic to the Splunk container through the `Logging Aggregator` setting. The playbook requires a variable called container_reference to be set. The container_reference variable needs to be how your AWX container will be able to talk to the Splunk container. See the SAML section for some examples for how to select a `container_reference`.
+
+Once you have your container reference you can run the playbook like:
+```bash
+export CONTROLLER_USERNAME=<your username>
+export CONTROLLER_PASSWORD=<your password>
+ansible-playbook tools/docker-compose/ansible/plumb_splunk.yml -e container_reference=<your container_reference here>
```

Comment:
I can't find this playbook `plumb_splunk.yml`

Reply:
Thanks, I missed it in the commit somehow, updated.